.github/workflows/build-push.yml

```diff
@@ -1,4 +1,4 @@
-name: Create and publish a Docker image
+name: Build and Push
```

Comment:
Build and Push Container Image?

Reply:
<img width="468" alt="Screenshot 2024-01-15 at 8 31 22 PM" src="https://github.com/substratusai/lingo/assets/10274189/32a26ea7-1712-4c13-866f-a64494e14b30">

Reply:
I am fine reverting, but the reason I made these changes: "Build and Push" as the title of the workflow and "container" as what is being built and pushed. I think it displays nicely on the output.